.editorconfig

```diff
@@ -1,8 +1,253 @@
 root = true
 
 [*]
-indent_style = space
-end_of_line = lf
 charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
 trim_trailing_whitespace = true
 insert_final_newline = true
+max_line_length = 100
```

Comment:
@michael-burwig can you confirm we want max_line_length here as 100 and not 120?